src/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js

```diff
@@ -159,7 +158,13 @@ export class ObjectsTable extends Component {
     // TODO: is there a good way to stop existing calls if the input changes?
     await ensureMinimumTime(
       (async () => {
-        const data = await savedObjectsClient.find({
+        const allSavedObjects = await savedObjectsClient.find({
+          type: INCLUDED_TYPES,
+        });
+
+        totalItemCount = allSavedObjects.total;
```

Comment:
Now that the totatItemCount ignores filters, does it still make sense to display right next to the page title? I find this a little misleading. How about moving the totalItemCount to "Export Everything" button since that is where the total number of objects is really only relevant.

Reply:
Good idea, will do.